frame/support/src/dispatch.rs

```diff
@@ -1446,25 +1444,14 @@ macro_rules! decl_module {
 					as
 					$system::Config
 				>::PalletInfo as $crate::traits::PalletInfo>::name::<Self>().unwrap_or("<unknown pallet name>");
-				let new_storage_version = $crate::crate_to_pallet_version!();
 
 				$crate::log::info!(
 					target: $crate::LOG_TARGET,
-					"⚠️ {} declares internal migrations (which *might* execute), setting storage version to {:?}",
```

Comment:
same note here about storage version being displayed is nice to have

Reply:
seems fixed